### PR TITLE
protobuf 3 support with backwards compatibility

### PR DESCRIPTION
**Pull request by clatko**
change the protobuf version in the pom from 2.6.1 to 3.0.0 and the tests will still pass.

### CODE REVIEW

**Comment by magro** [comment]
@clatko Thanks for the PR! Can you please elaborate a bit more the motivation for this?

**Comment by clatko** [comment]
This wil support protobuf 2 and 3. Just change the version in Pom.xml from 2.6.1 to 3.0.0 and all will work. Change back to 2.6.1 and it will still work. I didn’t have time to figure out how to test both versions simultaneously using the respective protobuf library versions.

**Comment by magro** [comment]
Let me rephrase my question: what does currently not work what is fixed with this PR?
And what is different to the recently merged PR #93?
TBB, I don't understand why github shows a diff at all...

**Comment by clatko** [comment]
It is actually the same thing. Hah. I'll close this one.

**Comment by magro** [comment]
Ok, thanks anyways :-)
